recipe/nvcc.profile.patch

```diff
@@ -1,12 +1,11 @@
 --- bin/nvcc.profile.bak	2024-03-12 15:52:23.511145665 +0000
 +++ bin/nvcc.profile	2024-03-12 16:27:28.806624738 +0000
-@@ -1,16 +1,16 @@
+@@ -1,15 +1,15 @@
  
 -TOP              = $(_HERE_)/..
 +TOP              = $(_HERE_)/../$(_TARGET_DIR_)
  
  CICC_PATH        = $(TOP)/nvvm/bin
- CICC_NEXT_PATH   = $(TOP)/nvvm-next/bin
```

Comment:
@robertmaynard am assuming this is ok since this is dropped from the source file. We were not patching this line. Though please let us know if we need to change anything here

Reply:
Ah missed Rob is out today. Think we should be ok here. So let's go ahead and merge

We can follow up with Rob when he get back